hack/README.md

```diff
@@ -19,38 +19,32 @@ diff --git a/hack/update-generated-code.sh b/hack/update-generated-code.sh
  #                  k8s.io/kubernetes. The output-base is needed for the generators to output into the vendor dir
  #                  instead of the $GOPATH directly. For normal projects this can be dropped.
  bash ${CODEGEN_PKG}/generate-groups.sh "deepcopy,client,informer,lister" \
--  github.com/kubernetes-csi/external-snapshotter/pkg/client github.com/kubernetes-csi/external-snapshotter/pkg/apis \
-+  github.com/kubernetes-csi/external-snapshotter/v2/pkg/client github.com/kubernetes-csi/external-snapshotter/v2/pkg/apis \
-   volumesnapshot:v1beta1 \
+   github.com/kubernetes-csi/external-snapshotter/client/v2 github.com/kubernetes-csi/external-snapshotter/client/v2/apis \
+  volumesnapshot:v1beta1 \
    --go-header-file ${SCRIPT_ROOT}/hack/boilerplate.go.txt
 ```
 
-Once you are done with patching, continue:
+Once you run the script, you got the output as:
 ```bash
-rm -fr v2/
-ln -sfvn $(pwd) v2
-rm -fr pkg/client
-```
-
-Run: ./hack/update-generated-code.sh from the project root directory.
+Generating deepcopy funcs
+Generating clientset for volumesnapshot:v1beta1 at github.com/kubernetes-csi/external-snapshotter/client/v2/clientset
+Generating listers for volumesnapshot:v1beta1 at github.com/kubernetes-csi/external-snapshotter/client/v2/listers
+Generating informers for volumesnapshot:v1beta1 at github.com/kubernetes-csi/external-snapshotter/client/v2/informers
 
-Do not forget to revert previously applied workaround:
-```bash
-git checkout -- v2/
-git checkout -- hack/update-generated-code.sh
 ```
 
+
 ## update-crd.sh
 
 This is the script to update CRD yaml files under ./config/crd/ based on types.go file.
 
-Make sure to run this script after making changes to /pkg/apis/volumesnapshot/v1beta1/types.go.
+Make sure to run this script after making changes to /client/apis/volumesnapshot/v1beta1/types.go.
 
 Follow these steps to update the CRD:
 
-* Run ./hack/update-crd.sh from root directory, new yaml files should have been created under ./config/crd/
+* Run ./hack/update-crd.sh from client directory, new yaml files should have been created under ./config/crd/
 
-* Replace `api-approved.kubernetes.io` annotation value in all yaml files in the metadata section with your PR.
+* Add `api-approved.kubernetes.io` annotation value in all yaml files in the metadata section with your PR.
```

Comment:
@xing-yang when i run the script "hack/update-crd.sh" in client directory the annotation `api-approved.kubernetes.io`
is not added there.So i modify it to Add this annotation rather replace.Could you provide some input on this. 
Thanks

Reply:
Yes, we should add it here.  Can you change to the following? 
* Add `api-approved.kubernetes.io` annotation value in all yaml files in the metadata section with the PR where the API is approved by the API reviewers. The current approved PR for snapshot beta API is `https://github.com/kubernetes-csi/external-snapshotter/pull/139`.  Refer to `https://github.com/kubernetes/enhancements/pull/1111` for details about this annotation.

Can you add it to the generated CRD manifests?  Please set it to `api-approved.kubernetes.io: "https://github.com/kubernetes-csi/external-snapshotter/pull/139"`.  The approved PR referenced in the existing code is wrong.